### PR TITLE
New package: shiromino-0.beta3.pre2

### PR DESCRIPTION
**Pull request by Francesco149**
MIT-licensed puzzle action game

kinda weird versioning, feel free to reject if it seems too unreliable

### CODE REVIEW

File: srcpkgs/shiromino/template [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Template file for 'shiromino'
+pkgname=shiromino
+version=0.beta3.pre2
```
Comment (maxice8) [inline srcpkgs/shiromino/template]: this version seems to be beta

File: srcpkgs/shiromino/template [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+revision=1
+_tag="v.beta3-pre2"
+wrksrc="${pkgname}-${_tag}"
+build_style=gnu-makefile
+hostmakedepends="pkg-config"
+makedepends="SDL2-devel SDL2_image-devel SDL2_mixer-devel"
+short_desc="Diverse and challenging puzzle/action game"
+maintainer="Franc[e]sco <lolisamurai@tfwno.gf>"
+license="MIT"
+homepage="https://github.com/FelicityVi/${pkgname}"
```
Comment (maxice8) [inline srcpkgs/shiromino/template]: write out the name completely here

File: srcpkgs/shiromino/template (hunk 1 above)
Comment (Francesco149) [reply srcpkgs/shiromino/template]: the only tags available are beta unfortunately.  feel free to reject if it's not good enough for packaging

**Comment by maxice8** [comment]
On Fri, Apr 12, 2019 at 04:33:30PM +0000, Franc[e]sco wrote:
> Francesco149 commented on this pull request.
> 
> 
> 
> > @@ -0,0 +1,32 @@
> +# Template file for 'shiromino'
> +pkgname=shiromino
> +version=0.beta3.pre2
> 
> the only tags available are beta unfortunately.  feel free to reject if it's not good enough for packaging
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/10862#discussion_r274980405

Sadly it is, i could accept a beta version that uses semver like 0.1.0

**Comment by Francesco149** [comment]
it's alright, i figured the version would be too weird

**Pull request closed without merging**